tsl/test/expected/custom_hashagg.out

```diff
@@ -0,0 +1,54 @@
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-TIMESCALE for a copy of the license.
+-- Test that the range estimation functions estimate_max_spread_var()
+-- is used for custom hash aggregates and that they behave in a sane
+-- manner when there are errors.
+--
+-- To trigger a call to the function, the following are required:
+-- 
+-- timescaledb.enable_custom_hashagg to be true
+-- 
+-- query should either date_trunc or time_bucket bucket function
+-- with a recognized time type
+-- 
+-- an aggregation function in the result
+-- 
+-- a group-by on the bucket created by the date_trunc or time_bucket
+-- 
+-- statistics recorded for the variable used in the bucketing
+-- function
+create table foo( a integer, b timestamptz);
+select count(*) from create_hypertable('foo', 'b');
+NOTICE:  adding not-null constraint to column "b"
+ count 
+-------
+     1
+(1 row)
+
+insert into foo values
+       (1, '2004-10-19 10:23:54'),
+       (1, '2005-10-19 10:23:54'),
+       (1, '2005-01-01 00:00:00+00'),
+       (2, '2005-01-01 00:00:00+00');
+set timescaledb.enable_custom_hashagg to true;
+analyze foo;
+set client_min_messages to debug1;
+select date_trunc('hour', b) bucket, sum(a) from foo group by bucket;
+LOG:  statement: select date_trunc('hour', b) bucket, sum(a) from foo group by bucket;
+DEBUG:  estimate_max_spread_var - min_datum: 151521834000000, max_datum: 183057834000000
```

Comment:
This printout is here just to check that we actually trigger execution of the function. We can probably remove it later but keep this test case.

Reply:
Hmm... it seems this function is not executed in release builds nor on MacOS, causing flakes. Might be better to remove it.

Reply:
I removed this debug printout. We have to trust codecov to check this.